### PR TITLE
fix: Reenable staleness bot

### PR DESCRIPTION
**Pull request by losalex**
Reenable staleness bot on the repo

### CODE REVIEW

File: owlbot.py [hunk 1]
```diff
@@ -21,5 +21,5 @@
 
 common_templates = gcp.CommonTemplates()
 templates = common_templates.node_library()
-s.copy(templates)
+s.move(templates, excludes=[".github/auto-label.yaml"])
```
Comment (minherz) [inline owlbot.py:24]: is `copy` a wrong way to operate the templates? why do you use `move()`?

**Review by minherz — APPROVED** [review]
i am concerned about replacing `copy()` with `move()`. what is a reason?

File: owlbot.py (hunk 1 above)
Comment (losalex) [reply owlbot.py:24]: Both works fine - just copied a line from other source with move

**Comment by losalex** [comment]
> i am concerned about replacing `copy()` with `move()`. what is a reason?

Copy/paste... Copy and move both works just same in this case

**Pull request merged**